### PR TITLE
Applied the translation API. 

### PR DESCRIPTION
**Pull request by keyuyan1145**
Add a dropdown option for user to view comments in different languages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 target/
 .idea
+credential.json
```
Comment (rjkroege) [inline .gitignore:3]: Very wise.

File: portfolio/src/main/webapp/experience.html [hunk 2]
```diff
@@ -1,98 +1,98 @@
 <!DOCTYPE html>
 <html>
 
-<head>
-	<meta charset="UTF-8">
-	<title>Experience</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="script.js"></script>
-</head>
+  <head>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/experience.html]: This is unnecessary whitespace change. Revert it please.

File: portfolio/src/main/webapp/project.html [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+      The form is a simple search box where the program searches through the back-end 
+      database for the user entered keyword. The search results are output to the client
+      in a table format.
+      </p>
+      <img class="projpic" src="images/mdb-lookup.PNG" alt="mdb-lookup results">
+      <img class="projpic" src="images/serverLog.PNG" alt="Server Log">
+    </div>
 
-</body>
+  </body>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/project.html]: Same comment as above. Unnecessary whitespace change. Please revert it.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -17,8 +17,8 @@
  */
 function randomFact() {
   const facts =
-      ['Sometimes I get my age wrong.', 'I\'m learning Chinese calligraphy.',
-       'I could speak 5 languages/dialects (technically).', 'I\'m a huge sports fan'];
+    ['Sometimes I get my age wrong.', 'I\'m learning Chinese calligraphy.',
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js:20]: Why the white space change?

**Review by rjkroege — COMMENTED** [review]
Please avoid unnecessary whitespace changes. Please consider making a separate PR for format changes, do it once and then maintain consistent formatting going forward.

**Comment by keyuyan1145** [comment]
Apologies. I have reverted the white-space commit, and I will have a separate PR for it at a later time.

**Review by rjkroege — APPROVED** [review]
Thanks for separating the whitespace changes.
